src/EA.Iws.Web/Areas/AdminExportNotificationMovements/Views/CaptureMovement/Create.cshtml

```diff
@@ -37,17 +39,61 @@
             <button class="button" type="submit">@Constants.ContinueButtonText</button>
 
         }
+        @Html.ActionLink(Resource.ReturnToSummary, "Index", "Home")
+        </div>
+        <br/>
+        <div class="column-one-third stickem" id="floatingSummary">         
+                <table>
+                    <thead>
+                        <tr>
+                            <th colspan="2" class="heading-small">@Model.NotificationNumber</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>@Resources.SummaryTitleIntended</td>
+                            <td>@Model.IntendedShipments</td>
+                        </tr>
+                        <tr>
+                            <td>@Resources.SummaryTitleNumber</td>
+                            <td>@Model.UsedShipments</td>
+                        </tr>
+                        <tr>
+                            <td>@Resources.SummaryTitleActive</td>
+                            <td>@Model.ActiveLoads</td>
+                        </tr>
+                        <tr>
+                            <td>@Resources.SummaryTitleReceived</td>
+                            <td>@Model.QuantityReceivedTotal</td>
+                        </tr>
+                        <tr>
+                            <td>@Resources.SummaryTitleRemaining</td>
+                            <td>@Model.QuantityRemainingTotal</td>
+                        </tr>
+                        <tr>
+                            <td>@Resource.AverageTonnage</td>
+                            <td>@Model.AverageTonnage</td>
+                        </tr>
+                    </tbody>
+                </table>          
+        </div>
     </div>
 </div>
 
-@Html.ActionLink(Resource.ReturnToSummary, "Index", "Home")
 
 @section scripts {
     @Html.Partial("_Scripts")
 
     <script>
         $(function() {
             $("#ShipmentNumber").focus();
-        })
+        });
     </script>
-}
\ No newline at end of file
+
+<script src="~/Scripts/jquery.stickem.js"></script>
```

Comment:
JavaScript files should be added to bundles (see BundleConfig.cs in the App_Start folder). That way it get's added with the existing scripts and minified.